src/cryptenroll/cryptenroll-wipe.c

```diff
@@ -357,9 +357,10 @@ int wipe_slots(struct crypt_device *cd,
 
         case WIPE_ALL:
                 r = find_all_slots(cd, wipe_slots, keep_slots);
-                if (r < 0)
+                if (r < 0){
+                        printf("You have deleted ALL slots, make sure to register new keys before rebooting");
```

Comment:
Again, this `if` block only hits on OOM.
If you want to suggest something, here is not a good place.